src/main/java/seedu/taassist/logic/commands/AddcCommand.java

```diff
@@ -76,14 +75,14 @@ public static String getCommandMessage(Set<ModuleClass> newClasses, Set<ModuleCl
 
     private static String getClassesAddedMessage(Set<ModuleClass> newClasses) {
         requireAllNonNull(newClasses);
-        String newClassesStr = newClasses.stream().map(Object::toString).collect(Collectors.joining(" "));
+        String newClassesStr = newClasses.stream().map(Object::toString).collect(Collectors.joining(", "));
```

Comment:
I believe this method is used quite commonly, perhaps we should add it to `StringUtil`.

Reply:
The problem of extracting this out is that not all messages call the `toString` method. For example, for `deletes`, it uses `Session::getSessionName`, `grade` uses `Student:getName` and `addc` uses `ModuleClass::toString`. I think it is doable only if we make the `toString` return the `name` of all the objects above.